flink-libraries/flink-cep/src/test/java/org/apache/flink/cep/nfa/NFAStateAccessTest.java

```diff
@@ -81,8 +81,8 @@ public void testComplexBranchingAfterZeroOrMore() throws Exception {
         nfaTestHarness.consumeRecords(inputEvents);
 
         assertEquals(58, sharedBuffer.getStateReads());
-        assertEquals(33, sharedBuffer.getStateWrites());
-        assertEquals(91, sharedBuffer.getStateAccesses());
+        assertEquals(41, sharedBuffer.getStateWrites());
```

Comment:
This change has been done to incorporate state writes being done by eventsCount.clear() since invoking mapstate clear method increments state writes by 1.